tests/Jellyfin.Dlna.Tests/ProfileTester.cs

```diff
@@ -0,0 +1,107 @@
+using Emby.Dlna;
+using Emby.Dlna.PlayTo;
+using MediaBrowser.Common.Configuration;
+using MediaBrowser.Controller;
+using MediaBrowser.Model.Dlna;
+using MediaBrowser.Model.IO;
+using MediaBrowser.Model.Serialization;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Jellyfin.Dlna.Tests
+{
+    public class ProfileTester
+    {
+        private DlnaManager GetManager()
+        {
+            var xmlSerializer = new Mock<IXmlSerializer>();
+            var fileSystem = new Mock<IFileSystem>();
+            var appPaths = new Mock<IApplicationPaths>();
+            var loggerFactory = new Mock<ILoggerFactory>();
+            var appHost = new Mock<IServerApplicationHost>();
+
+            return new DlnaManager(xmlSerializer.Object, fileSystem.Object, appPaths.Object, loggerFactory.Object, appHost.Object);
+        }
+
+        [Fact]
+        public void Test_Profile_Matches()
```

Comment:
No need to put `Test` in the method name, we know it's a test. We also try to adhere to the `MethodName_Scenario_ExpectedResult` naming scheme.
```suggestion
        public void IsMatch_GivenMatchingName_ReturnsTrue()
```